src/compiler/commandLineParser.ts

```diff
@@ -391,6 +391,12 @@ namespace ts {
             description: Diagnostics.Allow_default_imports_from_modules_with_no_default_export_This_does_not_affect_code_emit_just_typechecking
         },
         {
+            name: "strictESM",

```

Comment:
This is not a strict option per se.. this is an interop flag.. i would not use `strict` prefix since we have already given it a different meaning with `--strict`.

Reply:
🚲 🏠 🕐 Alright, `--standardESM` then?

Reply:
Spoke with daniel, gunna try `--ESMInterop`